compiler/damlc/daml-prim-src/GHC/Types.daml

```diff
@@ -136,6 +136,10 @@ data TyCon = TyCon Word# Word#           -- Fingerprint
                    KindRep               -- A representation of the type's kind
 
 
+-- | HIDE A DAML-LF call to an external package
+external : forall (f : Symbol) b. b
+external = external --deleted by the compiler
```

Comment:
I think we use `magic` for this in other places.

Reply:
I don't think so. This is analogous to how we handle `primitive`. I think `magic` is handled by ghc itself.

Reply:
Oh I see, thanks for the explanation.